Activity.js

```diff
@@ -39,7 +39,24 @@ class Activity {
     changeCardInfo(buttonName, cardStyle);
   }
 
-  saveToStorage() {
+  saveToStorage(pastActivities) {
 
+    // if (!localStorage) {
+      var stringified = JSON.stringify(pastActivities)
+      console.log("test2", pastActivities, stringified)
+      localStorage.setItem("pastActivitesKey", stringified)
```

Comment:
Still working on functionality to push > stringify> return correct pastActivities array; currently is only working on the first object, but creates a new ID with each push